### ISSUE TITLE
maintenancetool .new extension

### ISSUE THREAD

**Issue by DrMefistO** [Alexpux/MSYS2-packages#158]
maintenancetool.dat.new
maintenancetool.exe.new

After the new installation of MSYS2 x86_64 in my Windows 8.1 x64, I have this files instead of normal (maintenancetool.dat, maintenancetool.exe). After renaming .new extension, maintenance tool can't find any repo.

i686 version works good.

**Comment by Alexpux** [comment]
How you use maintenance tool?

**Comment by DrMefistO** [comment]
Just run and select Update components.

**Comment by Alexpux** [comment]
It will not update anything

**Comment by DrMefistO** [comment]
Ok, but what about .new extension?
Installer should remove it, but it don't.

**Comment by Alexpux** [comment]
How you get files with this extension?

**Comment by DrMefistO** [comment]
1) Run msys2-x86_64-20150202.exe;
2) Next, Next, Next;
3) ?????;
dev\
etc\
home\
mingw32\
mingw64\
tmp\
usr\
var\
autorebase.bat
components.xml
InstallationLog.txt
maintenancetool.ini
maintenancetool.dat.new
maintenancetool.exe.new
mingw32_shell.bat
mingw64_shell.bat
msys2.ico
msys2_shell.bat
network.xml

**Comment by Alexpux** [comment]
Do you have previous installations of MSYS2 before you run installer?

**Comment by DrMefistO** [comment]
No. It do not allow to install in the previous directory. So, previously, I have removed any installed version.

**Comment by Alexpux** [comment]
Probably Qt Framework Installer bug then.

**Comment by thomassisson** [comment]
This is a stupid, idiotic, but. What's the point of the maintenance tool if you can't use it. It should at least have a default repository.

**Comment by mati865** [comment]
It's used only for (un-) installation of MSYS2 so it doesn't need default repository.

**Comment by thomassisson** [comment]
Then what are the other options for?

Everything below is either replies, forwards, or advertising.

**Comment by mati865** [comment]
They are used to add, remove or update components of Qt SDK.
So they do nothing for MSYS2.

**Comment by DavidEGrayson** [comment]
> This is a stupid, idiotic, bug. What's the point of the maintenance tool if you can't use it. 

> Then what are the other options for?

MSYS2 is a free-time project run by a small group of volunteers.  They chose an open source installer framework that gets the job done (e.g. extracting files to a user-chosen directory, putting shortcuts in the Start Menu, and running some scripts afterwards).  Apparently it has some bugs, rough edges, and features we don't need, but that does not mean it's a stupid choice to use that tool.

Actually, if we are talking about the installer, a more pressing issue is that the current installer is from February and it installs an out-of-date version of pacman that requires the user to jump through some hoops to update the core packages like bash and msys2-runtime.  Alexey made a new installer on 2016-07-19 but he had to pull it because of some bug.
